### PR TITLE
Add fullbleed attribute to app-header-layout

### PR DESCRIPTION
**Pull request by blasten**
The idea is to have a `fullbleed` attribute similar to app-drawer-layout.

### CODE REVIEW

File: demo/demo2.html [hunk 1]
```diff
@@ -51,13 +51,13 @@
...
 
-  <app-drawer-layout>
+  <app-drawer-layout fullbleed>
 
     <app-drawer>
       <app-toolbar>App name</app-toolbar>
     </app-drawer>
 
-    <app-header-layout>
+    <app-header-layout fullbleed>
```
Comment (frankiefu) [inline demo/demo2.html]: Why are changing all the demos to use `fullbleed`?  Should we create new demos for showing `fullbleed` usage?

File: app-header-layout/app-header-layout.html [hunk 2]
```diff
@@ -91,10 +98,20 @@
...
         @apply(--layout-fit);
-
         overflow-y: auto;
         -webkit-overflow-scrolling: touch;
       }
+
+      :host([fullbleed]) {
+        @apply(--layout-vertical);
+        @apply(--layout-fit);
+      }
```
Comment (frankiefu) [inline app-header-layout/app-header-layout.html]: Let's move this css block to after `:host([has-scrolling-region])` so all the `:host` rules are kind of together.

Comment (blasten) [inline app-header-layout/app-header-layout.html]: done

File: demo/demo2.html (hunk 1 above)
Comment (blasten) [inline demo/demo2.html]: removed

**Pull request merged**